src/Manifest/Manifest.csproj

```diff
@@ -13,11 +13,20 @@
   <Import Project="..\..\build\NetFrameworkRelease.targets" />
 
   <ItemGroup>
-    <EmbeddedResource Include="$(MSBuildProjectDirectory)\..\bld\ReleaseInfo.json" />
+    <EmbeddedResource Include="$(TargetDir)ReleaseInfo.json" LogicalName="AccessibilityInsights.Manifest.ReleaseInfo.json" />
```

Comment:
The `LogicalName` is needed because the VS build changed the default resource name when the path changed. The client is hardcoded to use the original name, so we can't allow the name to change.